native/ios/SwrveUnitySDK/UnitySDK/UnitySwrveCommon.m

```diff
@@ -54,16 +54,32 @@ -(void) shutdown {
     _swrveSharedUnity = nil;
 }
 
-+(void) init:(char*)jsonConfig {
-    UnitySwrveCommonDelegate* swrve = [UnitySwrveCommonDelegate sharedInstance];
++(void) init:(char*)_jsonConfig {
+    NSString* jsonConfig = [UnitySwrveHelper CStringToNSString:_jsonConfig];
+    
+    NSString* spKey = @"storedConfig";
+    NSUserDefaults *preferences = [NSUserDefaults standardUserDefaults];
+    if((jsonConfig == nil) || (0 == [jsonConfig length])) {
+        jsonConfig = [preferences stringForKey:spKey];
+    }
+    if(jsonConfig == nil) {
```

Comment:
A length 0 string should never get into preferences, minimum should be "{}", but it's worth the check anyway.